### PR TITLE
Feature/ Action Items DAO-441 

### PR DESCRIPTION
**Pull request by Fabricevladimir**
- Action List Items
- Action Items
- Menu Items

### CODE REVIEW

File: packages/ui-components/src/components/menuItem/menuItem.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+   * Menu item text
+   */
+  label: string;
+
+  onClick?: () => void;
+};
+
+export const MenuItem: React.FC<MenuItemProps> = ({
+  icon,
+  isActive = false,
```
Comment (WissenIstNacht) [inline packages/ui-components/src/components/menuItem/menuItem.tsx]: If this is passed as prop, where is it evaluated?

File: packages/ui-components/src/components/menuItem/menuItem.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export type MenuItemProps = {
+  /**
+   * Icon to prepend to the Menu item text
+   */
+
+  icon: any; // TODO: set up proper type
```
Comment (WissenIstNacht) [inline packages/ui-components/src/components/menuItem/menuItem.tsx:9]: Did you see the file I sent on that topic?

**Comment by WissenIstNacht** [comment]
The buttons look nice! I left two comment on the menu item props.

File: packages/ui-components/src/components/menuItem/menuItem.tsx (hunk 1 above)
Comment (Fabricevladimir) [reply packages/ui-components/src/components/menuItem/menuItem.tsx]: `isActive` is used to determine whether to show the icon as well as the text and background colors for the container.

**Review by WissenIstNacht — APPROVED** [review]

**Review by sepehr2github — APPROVED** [review]

**Pull request merged**